Samples/DirectMLSuperResolution/DirectMLSuperResolution.vcxproj

```diff
@@ -98,7 +98,7 @@
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>WIN32;_DEBUG;_WINDOWS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <ConformanceMode>true</ConformanceMode>
```

Comment:
Probably a bunch of [C2102 ](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2102?view=msvc-160) errors? We should probably open a bug to address this. I'd be happy to clean this up.